package.json

```
@@ -42,8 +42,11 @@
     "build": "gatsby build",
     "build-ci": "gatsby build --prefix-paths",
     "start": "gatsby develop",
-    "format": "prettier --write '**/*.tsx'",
-    "test": "jest"
+    "format": "prettier --write **/*.{ts,tsx,js} !.cache !public",
+    "format-check": "prettier --check **/*.{ts,tsx,js} !.cache !public",
+    "jest": "jest",
+    "test": "yarn run jest && yarn run format-check && yarn run tslint",
```

CI showed that the files weren’t formatted.
Refs: 
https://travis-ci.com/nodejs/nodejs.dev/jobs/183197370

Maybe have a try to format directly when testing?


```suggestion
    "test": "yarn run jest && yarn run format && yarn run tslint",
```

---

I'm not a big fan of automatically changing code without user intervention. Afk right now but can take a peek into failure in the am

---

I fully agree with Myles. Because of reasons I explained here: https://github.com/nodejs/nodejs.dev/issues/164#issuecomment-469000093

---

Oh, I see. Thanks.